mat-process-api.Tests/V1/Controllers/ProcessDataImagesControllerTests.cs

```diff
@@ -420,16 +408,34 @@ public void when_GetProcessImage_controller_method_is_called_then_it_calls_the_l
         {
             //arrange
             var request = MatProcessDataHelper.CreateGetProcessImageRequestObject();
-            string expectedLogMessage = $"Get ProcessImage request for process reference: {request.processRef} and image Id: {request.imageId}";
+            string expectedLogMessage = $"Get ProcessImage request for Process Type: {request.processType}, Process Reference: {request.processRef}, Image Id: {request.imageId} and File Extension: {request.fileExtension}";
+
+            //act
+            try { _processImageController.GetProcessImage(request); } catch(Exception ex) { } //the empty try-catch is needed to ignore the exception being thrown due to absense of validation result returned later down the line. The idea is that logger is the first thing that is being called once request comes in.
 
-            _mockGetValidator.Setup(l => l.Validate(It.IsAny<GetProcessImageRequest>())).Returns(new FV.ValidationResult()); //setup validator to return a no error validation result
+            //assert
+            _mockLogger.Verify(l => l.Log(
+                        LogLevel.Information,
+                        It.IsAny<EventId>(),
+                        It.Is<FormattedLogValues>(v => v.ToString().Contains(expectedLogMessage)),
+                        It.IsAny<Exception>(),
+                        It.IsAny<Func<object, Exception, string>>()
+                    ), Times.AtLeastOnce);
+        }
+
+        [Test]
+        public void given_a_request_with_null_properties_when_GetProcessImage_controller_method_is_called_then_it_makes_the_logger_display_those_null_properties_as_string_saying_null_during_the_initial_logger_call_independent_of_the_validator_behaviour()  //Since there's no validator setup, it returns no validation result. If the Verify bit shows that logger was called under these conditions, then this shows that validation result has no influence on logger being called at least once, which is important if you want to register that request happened regardless of the validity of the request.
+        {
+            //arrange
+            var request = new GetProcessImageRequest(); //request with null properties
+            string expectedLogMessage = $"Get ProcessImage request for Process Type: {"null"}, Process Reference: {"null"}, Image Id: {"null"} and File Extension: {"null"}";
 
             //act
-            _processImageController.GetProcessImage(request);
+            try { _processImageController.GetProcessImage(request); } catch (Exception ex) { } //the empty try-catch is needed to ignore the exception being thrown due to absense of validation result returned later down the line. The idea is that logger is the first thing that is being called once request comes in.
```

Comment:
You should try to mock the validation result, rather than using a try/catch within a test.

Reply:
Not mocking the validation result is the entire idea behind this test. I explained it in the comment next to the test, and in the commit message in bit more detail.
If I mock the result then I'll change what the test is asserting.

Reply:
Please correct me if I am wrong, but the test aims at verifying that the logger will log the first message, even if the values passed are null? The logger should be called after the logging message, which means that the message should be logged irrespective to the validation result (positive or not). If that is the case, you can still mock the validation result and test that the logger logged the first line.

Reply:
The test aims to show that the request gets logged at least once regardless of validator behaviour.
By behaviour I mean, it shouldn't matter whether the validator returned true or false, or if it crashed entirely.
I think that if your initial log is dependent on the validator not crashing, then it's a bad of way of logging that request happened.
So I set up the test in such a way, that doesn't have validator implementation or setup to show that despite the validator crash (which happened in the try-catch block), the logger was indeed called at least once.

Reply:
I'll update the comments next to the test to make this clearer. I can see where the confusion is coming from now that I read it again. 

Sorry, my bad. initially I thought it was pretty good comment, but forgot to clarify the bit about crashing the flow being necessary (As there's no better way test this, unless Moq in the future implements 'VerifyThatMethodWasCalledBefore(...)') - only did that in the commit message.

Reply:
Added better description now above the 2 tests.